### PR TITLE
custom execute, more commands

### PR DESCRIPTION
**Pull request by Chive**

### CODE REVIEW

File: aldryn_client/localdev/main.py [hunk 1]
```diff
@@ -23,110 +24,64 @@ def docker_compose(*commands):
...
             clone_args.append(path)
-        subprocess.check_output(clone_args, stderr=subprocess.STDOUT)
+        execute(clone_args, silent=False, stderr=subprocess.STDOUT)
     except subprocess.CalledProcessError as exc:
         raise click.ClickException(exc.output)
 
-    click.echo('\nCreating workspace...')
+    # Detect old style or invalid projects
+    if not os.path.isfile(os.path.join(path or website_slug, 'docker-compose.yml')):
+        import ipdb; ipdb.set_trace()
```
Comment (czpython) [inline aldryn_client/localdev/main.py]: huh?

Comment (Chive) [inline aldryn_client/localdev/main.py]: thank you.

File: aldryn_client/localdev/main.py [hunk 2]
```diff
@@ -135,11 +90,83 @@ def create_workspace(client, website_slug, path=None):
...
+    ], silent=True, stderr=subprocess.STDOUT)
+
+    click.secho('inserting database dump', fg='green')
+    # FIXME: because of different ownership,
+    # this spits a lot of warnings which can
+    # ignored but we can't really validate success
+    with dev_null() as devnull:
+        try:
+            piped_restore = (
+                'tar -xzOf /app/{}'
```
Comment (czpython) [inline aldryn_client/localdev/main.py]: if I'm not mistaken pg_restore can actually restore from a .tar file

**Comment by czpython** [comment]
Ok to merge

**Pull request merged**